<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncomeExpense</class>
 <widget class="QMainWindow" name="IncomeExpense">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>693</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Income/Expense Tracker</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="income_description">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="income_source">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Job</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Parents</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Scholarships</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Student Aid</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="date_income">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy/MM/dd</string>
    </property>
    <property name="date">
     <date>
      <year>2024</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="income_amount">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="expense_description">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="expense_category">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>170</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Food</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Entertainment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Travelling</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clothing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Educational Equipment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>University Fees</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="dateEdit_expense">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy/MM/dd</string>
    </property>
    <property name="date">
     <date>
      <year>2024</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="expense_amount">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="add_income">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Income</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_expense">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Expenses</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_income">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>421</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_expenses">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>390</y>
      <width>411</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Source :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Date :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount (Rs.) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>169</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Category :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>49</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Date :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount (Rs.) :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BudgetButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>620</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add a Budget</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>620</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Financial Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bargraphbutton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>620</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Comparison</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endButtton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>620</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Ravie&quot;;</string>
    </property>
    <property name="text">
     <string>Welcome To Income / Expense Tracker</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;The Last Shuriken&quot;;
text-decoration:underlined;</string>
    </property>
    <property name="text">
     <string>For Income :-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;The Last Shuriken&quot;;</string>
    </property>
    <property name="text">
     <string>For Expenses :-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
